src/main/java/com/depromeet/todo/application/MockTokenService.java

```diff
@@ -0,0 +1,22 @@
+package com.depromeet.todo.application;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class MockTokenService implements TokenService<Long> {
+    @Override
+    public String generate(Long memberId) {
+        return null;
+    }
+
+    @Override
+    public Long decode(String token) {
+        if (token == null) {
+            return null;
+        }
+        if (token.equals("todoAccessToken")) {
```

Comment:
```suggestion
        if ("todoAccessToken".equals(token)) {
```

로 하면 Null Check를 안해도 될 것 같아요!

Reply:
감사합니당! 말씀듣고보니 바보같은 코드였네요 ㅋㅋㅋㅋ